### ISSUE TITLE
Add User Notifications for High Profile Actions

### ISSUE THREAD

**Issue by ssmereka** [smartdevicelink/sdl_shaid#70]
When an application ID is revoked, deleted, or other important actions take place, a user should be notified.  Currently notifications occur via email, if the user has one registered.

**Comment by CMaylone** [comment]
Going to build a generic notification for all of these things.

**Comment by CMaylone** [comment]
@ssmereka Added functionality for sending a very generic email. Not sure where the triggers should be.
```javascript
var notification = new (require(require('path').resolve('./libs/notification/')))(config, log);
notification.sendGenericEmail('my.subject.translation.string', 'my.content.translation.string', 'my.alert-header.translation.string', 'alert-good', cb)
```

**Comment by ssmereka** [comment]
Fixed in commit 171e40c
